gpytorch/lazy/lazy_tensor.py

```diff
@@ -1073,6 +1073,11 @@ def root_decomposition(self):
                 "Got a {} of size {}.".format(self.__class__.__name__, self.size())
             )
 
+        # when dealing with small matrices, it's usually faster to use Choleksy decomposition
+        if self.numel() < 250:
```

Comment:
```suggestion
        if self.matrix_shape.numel() < 256:
```

Reply:
This will work for large batch matrices, and 256 for supporting powers of 2.

Reply:
Actually, rather than using a magic number we should maybe hardcode this into a setting.